src/main/java/org/example/workerapi/entity/Department.java

```diff
@@ -0,0 +1,23 @@
+package org.example.workerapi.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@RequiredArgsConstructor
+public class Department {
+
+    private String title;
+    @Id
+    private Long managerPersonalCode;
+    @OneToMany(mappedBy = "department",cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Šitoj vietoj rekomenduočiau padaryti paprastą generuojamą ID, o šitą managerPersonalCode kaip paprastą lauką